Bynder/Api/Queries/MediaQuery.cs

```diff
@@ -42,6 +42,12 @@ public class MediaQuery
         public string Keyword { get; set; }
 
         /// <summary>
+        /// The type of the asset
+        /// </summary>
+        [APIField("type")]
+        public string Type { get; set; }
```

Comment:
Ok, I changed it as you suggested and made a new push.
Shouldn't we also change Media.Type to be AssetType instead of string, however this is a breaking change, so existing users should be warn about it.